resources/skins/default/1080i/script-upnext-stillwatching-simple.xml

```diff
@@ -41,7 +41,7 @@
                         <control type="button" id="10">
                             <label>    $ADDON[service.upnext 30010]</label>
                             <onclick>SendClick(3012)</onclick>
-                            <visible>Integer.IsGreater(Player.TimeRemaining,59)</visible>
+                            <visible>Integer.IsGreater(Window.Property(remaining),59)</visible>
```

Comment:
Was this a bug, or is this necessary for something else?

Reply:
It will only become a bug if the functionality in PR#200 is desired to be implemented.

Normally the popup remaining time is the same as Player.TimeRemaing, but not if the popup is meant to do something before the end of the file.